USER_MANUAL.md

```diff
@@ -336,6 +363,345 @@ _TODO_
+
+**Notes on acquisition:** Refer to the section [Virtual World Defining Files](#10-virtual-world-defining-files) for comprehensive instructions on setting up this file.
+
+**Notes of caution:** The site types defined in this file must correspond to the values in the [sites_file](#sites_file). Refer to the [Virtual World Defining Files](#10-virtual-world-defining-files) for more details.
+
+#### &lt;equipment_group_file&gt;
+
+**Data Type:** String
+
+**Default input:** None
+
+**Description:** This parameter is defined by a string specifying the name of the CSV file containing data on the equipments in the simulation. It's an _optional_ file designed to refine site characteristics and reduce the need for redundant definitions of equipment groups.
+
+**Notes on acquisition:** Refer to the section [Virtual World Defining Files](#10-virtual-world-defining-files) for comprehensive instructions on setting up this file.
+
+**Notes of caution:** The equipment defined in this file must correspond to equipment defined in the [sites_file](#sites_file). Refer to the [Virtual World Defining Files](#10-virtual-world-defining-files) for more details.
+
+#### &lt;sources_file&gt;
+
+**Data Type:** String
+
+**Default input:** None
+
+**Description:** This parameter is determined by a string indicating the name of the CSV file containing information about the components and sources in the simulation. It's an _optional_ file that enables users to provide more detailed data regarding individual components and the types of sources associated with each component.
+
+**Notes on acquisition:** Refer to the section [Virtual World Defining Files](#10-virtual-world-defining-files) for comprehensive instructions on setting up this file.
+
+**Notes of caution:** The component and sources must correspond to the values found in the [equipment_group_file](#equipment_group_file) and [emissions_file](#emissions_file). Refer to the [Virtual World Defining Files](#10-virtual-world-defining-files) for more details.
+
+### &lt;site_samples&gt;
+
+**Data Type:** Integer
+
+**Default input:** None
+
+**Description:** This variable is a integer indicating the number of sites to subset.
+
+**Notes on acquisition:** No data acquisition required.
+
+**Notes of caution:** The number of site sampled must be equal to or less than the number of unique sites provided in the [sites_file](#sites_file).
+
+### &lt;consider_weather&gt;
+
+**Data type:** Boolean
+
+**Default input:** False
+
+**Description:** Specify if the weather envelopes will be considered when determining the different methods work hours per each day.
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** Even if weather is considered false, [weather_file](#weather_file) must be present and valid.
```

Comment:
Is this still true?

Reply:
Yes. This is on the list of TODO, to make sure that if weather file is not used, that it shouldn't be a required input.